front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/AnalyticsChartWidget/index.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+
+// Components
+import { Box, Input, Title } from '@citizenlab/cl2-component-library';
+import GenderChart from 'containers/Admin/dashboard/users/Charts/GenderChart';
+import AgeChart from 'containers/Admin/dashboard/users/Charts/AgeChart';
+import VisitorsCard from '../../../../../../../modules/commercial/analytics/admin/components/VisitorsCard';
```

Comment:
Hmm we should not be importing things from modules. If you don't use all the `../` you should get a linter error for that. Forgot about that... maybe we should consider moving (some parts of) `analytics` to core, or at least have that conversation with Iva and/or Koen G

Reply:
Yes I wanted to talk to you about that - I was getting a linter error with these. Understand why, but we need to decide what to do with it.

Reply:
I have created https://citizenlab.atlassian.net/browse/CL-2281 to address this issue later